Playground/js/main.js

```diff
@@ -164,7 +164,12 @@ compileAndRun = function (parent, fpsLabel) {
                     sceneToRender.render();
                 }
 
-                fpsLabel.innerHTML = engine.getFps().toFixed() + " fps";
+                // Update FPS if camera is not a webxr camera
+                if(!(sceneToRender.activeCamera && 
+                    sceneToRender.activeCamera.getClassName && 
+                    sceneToRender.activeCamera.getClassName() === 'WebXRCamera')) {

```

Comment:
I would have expected a !== ?

Reply:
this is the condition of an existing active camera, and that it is an xr camera.

If the scene has activeCameras (plural) a `!==` would return false, since I am checking for activaCamera existence

Reply:
ok just me being dumb (not seeing the ! at the beginning ;))